support-frontend/assets/helpers/forms/paymentIntegrations/oneOffContributions.ts

```diff
@@ -150,13 +176,15 @@ const postOneOffAmazonPayExecutePaymentRequest = (data: AmazonPayData) =>
 const processStripePaymentIntentRequest = (
 	data: CreateStripePaymentIntentRequest,
 	handleStripe3DS: (clientSecret: string) => Promise<Stripe3DSResult>,
-) =>
+): Promise<PaymentResult> =>
 	handleOneOffExecution(
 		postToPaymentApi(data, '/contribute/one-off/stripe/create-payment').then(
 			(createIntentResponse) => {
-				if (createIntentResponse.type === 'requiresaction') {
+				const _createIntentResponse =
```

Comment:
Why the use of the proceeding underscore in this variable name?

Reply:
Created an intermediary variable to type cast the parameter and went with an underscore as I noticed that convention used elsewhere in the codebase